cypress/integration/common/support/textToId.js

```diff
@@ -9,6 +9,8 @@ export const textToId = {
   "i'm not sure": 'in-hiding-not-sure',
   "no i'm not worried about searching in public": 'in-hiding-no',
   'one to three countries': 'where-country',
+  "i'm not sure they could be anywhere": 'where-not-sure',
+  'make an enquiry': 'make-an-enquiry-button',
```

Comment:
I don't want to sound like a fusspot, but can I draw your attention to line 1 of this file ;-)

Reply:
done